packages/runtime/src/next/components/tests/makeswift-component-rendering.test.tsx

```diff
@@ -1,106 +1,117 @@
 /** @jest-environment jsdom */
 
+import { Suspense, type ComponentType } from 'react'
 import '@testing-library/jest-dom'
 import { act } from 'react-dom/test-utils'
 import { render, screen } from '@testing-library/react'
+
 import { ReactRuntime } from '../../../react'
 import * as Testing from '../../testing'
 import { MakeswiftComponent } from '../../../runtimes/react/components/MakeswiftComponent'
-import {
-  type MakeswiftComponentSnapshotMetadata,
-  type MakeswiftComponentDocument,
-  type MakeswiftComponentDocumentFallback,
-  type MakeswiftComponentSnapshot,
-} from '../../../client'
-import { type CacheData } from '../../../api/react'
-import { TextInput } from '@makeswift/controls'
+import { type MakeswiftComponentSnapshot } from '../../../client'
```

Comment:
Reworked/cleaned up components rendering tests to use already existing testing utilities.